src/test/ut/ut_cortxfs_xattr_ops.c

```diff
@@ -408,11 +408,16 @@ int listxattr_test_setup(void **state)
 	struct ut_xattr_env *ut_xattr_obj = XATTR_ENV_FROM_STATE(state);
 	struct ut_cfs_params *ut_cfs_obj = &ut_xattr_obj->ut_cfs_obj;
 
-	ut_xattr_obj->xattr[0] = file_name;
+
+	ut_xattr_obj->xattr = malloc(sizeof(char) * xattr_set_cnt+1);
```

Comment:
This will be char* to reflect that this is an array of pointer, not a string. This holds an array of strings.

Reply:
fixed.